cockatrice/src/setsmodel.h

```diff
@@ -43,11 +43,11 @@ class SetsModel : public QAbstractTableModel {
     QStringList mimeTypes() const;
     void swapRows(int oldRow, int newRow);
     void toggleRow(int row, bool enable);
-    void toggleAll(bool enable);
+    void toggleRow(int row);
+    void toggleAll(int row);
```

Comment:
I do't understand what this interface means? I toggle all sets but I'm telling it a row? Can you help me understand the intent here?